kedro_azureml/generator.py

```diff
@@ -191,12 +195,30 @@ def _construct_azure_command(
             environment=self._resolve_azure_environment(),  # TODO: check whether Environment exists
             inputs={
                 self._sanitize_param_name(name): (
-                    Input(type="string") if name in pipeline.inputs() else Input()
+                    Input(type="string")
```

Comment:
Let's add a comment for next generations of contributors that this condition is for "root" datasets (first nodes in the pipeline)

Reply:
This is now contained in the name of the helper function. Would you still like me to add a comment?